pkg/machinery/config/configdiff/configdiff.go

```diff
@@ -0,0 +1,129 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package configdiff provides a way to compare two config trees.
+package configdiff
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/hexops/gotextdiff"
```

Comment:
> This repository has been archived by the owner on Feb 9, 2024. It is now read-only.

Can't we use https://github.com/sergi/go-diff?

Reply:
I would rather use this one, we already use it in Omni without issues.

Reply:
And it doesn't support unified diffs, so it's useless here